api/tests/devcomp/unit/domain/usecases/record-passage-events_test.js

```diff
@@ -56,12 +60,29 @@ describe('Unit | Devcomp | Domain | UseCases | record-passage-events', function
       type: 'FLASHCARDS_RETRIED',
     };
 
+    const startedEvent = {
+      contentHash: Symbol('contentHash'),
+      occurredAt: new Date(),
+      passageId: 2,
+      sequenceNumber: 1,
+      type: 'PASSAGE_STARTED',
+    };
+
+    const terminatedEvent = {
+      occurredAt: new Date(),
+      passageId: 2,
+      sequenceNumber: 1,
+      type: 'PASSAGE_TERMINATED',
+    };
+
     const events = [
       flashcardsVersoSeenEvent,
       flashcardsStartedEvent,
       flashcardsCardAutoAssessedEvent,
       flashcardsRectoReviewedEvent,
       flashcardsRetriedEvent,
+      startedEvent,
```

Comment:
```suggestion
passageStartedEvent
```
Pour le différencier de d'autres "startedEvent" qui pourraient exister (ex. il y en a un pour les flashcards qu'on a préfixé, ligne 80)